src/components/player-info.js

```diff
@@ -66,19 +84,133 @@ AFRAME.registerComponent("player-info", {
     deregisterComponentInstance(this, "player-info");
   },
   play() {
+    ["happy", "sad", "eww", "disgust", "angry", "smile", "hearts", "surprise", "clean"].map(x =>
+      this.el.sceneEl.addEventListener(x, this.emojiAction(x))
+    );
     this.el.addEventListener("model-loaded", this.applyProperties);
     this.el.sceneEl.addEventListener("presence_updated", this.updateDisplayName);
     if (this.isLocalPlayerInfo) {
       this.el.querySelector(".model").addEventListener("model-error", this.handleModelError);
     }
   },
   pause() {
+    ["happy", "sad", "eww", "disgust", "angry", "smile", "hearts", "surprise", "clean"].map(x =>
+      this.el.sceneEl.removeEventListener(x, this.emojiAction(x))
+    );
     this.el.removeEventListener("model-loaded", this.applyProperties);
     this.el.sceneEl.removeEventListener("presence_updated", this.updateDisplayName);
     if (this.isLocalPlayerInfo) {
       this.el.querySelector(".model").removeEventListener("model-error", this.handleModelError);
     }
   },
+
+  emojiAction(type) {
+    switch (type) {
+      case "happy":
+        return this.changeHappyEmoji;
+      case "sad":
+        return this.changeSadEmoji;
+      case "eww":
+        return this.changeEwwEmoji;
+      case "disgust":
+        return this.changeDisgustEmoji;
+      case "angry":
+        return this.changeAngryEmoji;
+      case "smile":
+        return this.changeSmileEmoji;
+      case "hearts":
+        return this.changeHeartsEmoji;
+      case "surprise":
+        return this.changeSurpriseEmoji;
+      case "clean":
+        return this.cleanEmoji;
+    }
+  },
```

Comment:
Instead of mapping from an emoji type to a function, you could just map from the type directly to the image with a simple javascript object:
```javascript
const emojiTypeToImage = {
  happy: happyEmoji,
  sad: sadEmoji,
  ...
};
```
and then just access the image with `emojiTypeToImage[emojiType]`
That way, you wouldn't have to have all the individual `changeHappyEmoji`, etc. functions below.